usaspending_api/references/serializers.py

```diff
@@ -132,9 +132,11 @@ class Meta:
         default_fields = [
             "legal_entity_id",
             "parent_recipient_unique_id",
+            "recipient_unique_id",
```

Comment:
Thanks for catching this. Oof, looks like another piece of confusion re: our legacy data models (_i.e._, `legal_entity_id` as the surrogate key). Out of scope for this work, but just noting since I thought your CFDA work was the end of our FK-renaming days. I added this note to our db model cleanup story.